File: cs50/PSET1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int h;
int main(void)
{
    do
    {
        // ask user for height
        h = get_int("Hight: ");
    }
    // the hight should be >= 1
    while (h <= 0);
    // the space between each line
    for (int i = 0; i < h; i++)
    {
        // the space before each cube
        for (int k = 0; k < h - (i + 1); k++)
        {
            printf(" ");
        }
        // print each cube
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: cs50/PSET6/sentimental-mario-less/mario.py
# import get_int
from cs50 import get_int

# the integer must be between 1 and 8 inclusive
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# print the pyramid
for i in range(n):
    print(" " * (n - (i + 1)) + "#" * (i + 1))
